### DISCUSSION TITLE
[FEATURE REQ] Deluge add option "move completed to"

### DISCUSSION THREAD

**Discussion by nalsai** [autobrr/autobrr#1692] [Ideas]
**Version**
v1.42.0

**Describe the bug**
I'm using Deluge 2.1.1 and have set the download path in deluge to "/data/incomplete" with "move completed to" checked and set to "/data/completed".
Using autobrr with a save path set, the file gets downloaded to the save path but moved to "/data/completed" once complete.

**To Reproduce**
1. In Deluge: enable "move completed to" and set a path
2. Download something using autobrr with save path set in the deluge download action
3. Check the path of the file in deluge / your filesystem

**Expected behavior**
I would expect the file to be downloaded to my download path and then moved to the save path once the download has completed. (This is how for example FlexGet does it.)
Alternatively, downloading directly to the save path and disabling "move completed to" would be acceptable too.

**Desktop (please complete the following information):**
 - OS: Fedora 40
 - Browser Firefox 126

**Comment by zze0s** [comment]
Hi! I think this is the default behavior but we can pass `MoveCompleted bool` to disable/enable this and `MoveCompletedPath string` to specify a custom path to be moved to, so then Save Path becomes the incomplete location, I think.

We'd have to add that to the action settings for Deluge.
